homepage/homepage_updater.go

```diff
@@ -78,11 +79,11 @@ func (hu *HomepageUpdater) GetHomePageUpdateFor(ctx context.Context, userAccessT
 			imageObjects := map[string]image.ImageDownload{}
 			for _, fc := range homepageContent.AroundONS {
 				if fc.ImageID != "" {
-					image, err := hu.clients.ImageAPI.GetDownloadVariant(ctx, userAccessToken, "", "", fc.ImageID, ImageVariant)
+					imageDetails, err := hu.clients.ImageAPI.GetDownloadVariant(ctx, userAccessToken, "", "", fc.ImageID, ImageVariant)
```

Comment:
Agree with renaming from `image` to avoid a clash with the package name, but 'd suggest `imageVariant` as we are using `GetImageVariant`
```suggestion
					imageVariant, err := hu.clients.ImageAPI.GetDownloadVariant(ctx, userAccessToken, "", "", fc.ImageID, ImageVariant)
```
NB. This will probably result in a clash with the const `ImageVariant` but that in turn would be better named `ImageVariantName` as it's more representative of what it is.